src/views/Nft/market/Profile/components/UserNfts.tsx

```diff
@@ -21,11 +24,16 @@ interface SellNftProps {
 }
 
 const UserNfts = () => {
-  const { nfts, userNftsInitializationState } = useUserNfts()
+  const { account } = useWeb3React()
+  const { isLoading: isProfileLoading, profile } = useProfile()
+  const { lastUpdated, setLastUpdated: refresh } = useLastUpdated()
+  const { nfts, isLoading: isNftLoading } = useNftsForAddress(account, profile, isProfileLoading, lastUpdated)
```

Comment:
`useNftsForAddress` is used in ConnectedProfile/UnconnectedProfile and here
This will fetch twice and have two same state

Reply:
Done